dspace-rest/src/main/java/org/dspace/rest/common/HierarchyRepository.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.rest.common;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * Used to handle/determine status of REST API.
+ * Mainly to know your authentication status
+ *
+ */
+@XmlRootElement(name = "repository")
+public class HierarchyRepository extends HierarchyCommunity
```

Comment:
This is a minor point, but I'm wondering if we'd rather name this Site, instead of Repository... We have a DSO for Sites, but nothing for repository. So, perhaps it would be better to rename this to keep our naming consistent. Although we currently don't do anything with Sites, that could change...